reflex/compiler/utils.py

```diff
@@ -324,6 +319,17 @@ def get_theme_path() -> str:
     return os.path.join(constants.WEB_UTILS_DIR, constants.THEME + constants.JS_EXT)
 
 
+def get_root_stylesheet_path() -> str:
```

Comment:
This could be defined in `constants.py`

Reply:
Yeah I thought so but I had it here for consistency with other path functions